vumi_wikipedia/tests/test_text_manglers.py

```diff
@@ -69,3 +69,64 @@ def test_format_more(self):
                          fmt(long_ascii(), 4))
         self.assertEqual((55, u'...' + long_unicode(14, u' ... (more)')),
                          fmt(long_unicode(), 4))
+
+    def test_format_very_long_words(self):
+        """
+        If we have a very long word at the start of our input, we split it in
+        the middle.
+        """
+        cf = ContentFormatter(160, 70)
+
+        long_ascii_words = u' '.join([u'abc' * 60] * 2)
+        long_unicode_words = u' '.join([UNI_BIT * 25] * 2)
+
+        self.assertEqual(
+            long_ascii_words[:156] + u' ...', cf.format(long_ascii_words))
+        self.assertEqual(
+            long_unicode_words[:66] + u' ...', cf.format(long_unicode_words))
+
+        self.assertEqual(
+            long_ascii_words[:146] + u' ... (postfix)',
+            cf.format(long_ascii_words, u' (postfix)'))
+        self.assertEqual(
+            long_unicode_words[:56] + u' ... (postfix)',
+            cf.format(long_unicode_words, u' (postfix)'))
+
+    def test_format_more_very_long_words(self):
+        """
+        If we have a very long word at the start of our input, we split it in
+        the middle.
```

Comment:
The docstring is accurate, we're testing with multiple chunks of content rather than just the first. We still only split a long word if it's at the start of the input, although that may not be the start of the content.